RichEditorView/Assets/editor/style.css

```diff
@@ -22,13 +22,19 @@ html {
 
 body {
     min-height: 100%;
-    overflow: hidden;
+    overflow: auto;
 }
 
 body, h1, p {
     margin: 0;
 }
 
+div {
+    display: block;
+    position: static;
+    min-height: 28px;
```

Comment:
What is this 28px about? Is it necessary? I actually don't know the implications of much of this CSS that's being added. I hope it doesn't change the way it looks for people who are already using it.